go/vt/vttablet/tabletmanager/vreplication/engine_test.go

```diff
@@ -154,6 +154,7 @@ func TestEngineExec(t *testing.T) {
 
 	dbClient.ExpectRequest("use _vt", &sqltypes.Result{}, nil)
 	dbClient.ExpectRequest("insert into _vt.vreplication values(null)", &sqltypes.Result{InsertID: 1}, nil)
+	dbClient.ExpectRequest("select @@auto_increment_increment", &sqltypes.Result{}, nil)
```

Comment:
Also in these places, I think it's worth being explicit about the desired variable scope (assuming it's @@session).